lib/matplotlib/backend_bases.py

```diff
@@ -1633,6 +1633,7 @@ def __init__(self, figure=None):
         # We don't want to scale up the figure DPI more than once.
         figure._original_dpi = figure.dpi
         self._device_pixel_ratio = 1
+        super().__init__()  # Typically the GUI widget init (if any).
```

Comment:
Do we want to pass `*args, **kwargs` through here as well?

Reply:
`FigureCanvasBase.__init__` itself doesn't take `*args, **kwargs`; let's not bother adding that unless there's a real reason to do so.

Reply:
I am weakly in favor of atleast `**kwargs` flowing through this init, but also fine with waiting until it actually is a need someone has.

Reply:
I think passing `**kwargs` through only makes sense if `figure` was kwonly as well (so that at least in theory, each class in the inheritance tree can pick out whatever kwargs they support).  Once you start having positional arguments, the scheme breaks down anyways (as you can't tell which positional arguments you need to pick out).